scripts/ingest_dfc/db_utils.py

```diff
@@ -0,0 +1,16 @@
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from app.core import config
+
+
+engine = create_engine(
+    config.SQLALCHEMY_DATABASE_URI,
+    pool_pre_ping=True,
+    # TODO: configure as part of scaling work
+    pool_size=10,
+    max_overflow=240,
+)
+
+
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine, expire_on_commit=False)
```

Comment:
We've only changed this for the script for now, but this code should be forming the basis of a new ingest endpoint, so I think we need to be very careful modifying session defaults around expiry.

Reply:
Now removed